### DISCUSSION TITLE
Flutter SDK ( stream_chat_flutter: ^2.0.0-nullsafety.7) ChannelList View bug

### DISCUSSION THREAD

**Discussion by ghost** [GetStream/stream-chat-flutter#490] [Q&A] [answered]
Hey there devs, I am new to this SDK. I am not sure how to solve this bug
![image](https://user-images.githubusercontent.com/34367036/122883802-29b2cb00-d35b-11eb-92e9-550126d1c979.png)
Any help will be appreciated :)

**Comment by rlee1990** [comment, marked as answer]
@ROCKETSINGH6 you want to set filters like this: 
```
filter: Filter.in_('members', [idUser]),
```
Filter.in is for checking if any of the values in an array match.

**Reply by ghost** [reply]
Thanks! This worked!
